cleverpush/src/main/java/com/cleverpush/CleverPush.java

```diff
@@ -1022,8 +1023,10 @@ public void onConnected(@Nullable Bundle bundle) {
                                 geofencePendingIntent
```

Comment:
L1008 where you call addGeofences
maybe do it like here and see if it helps
so add the pendingIntent to the parameters of addGeofences and add the success and failurelistener and ofc some logs inside those
@ozamanav  @jprajapati1996 

![image](https://user-images.githubusercontent.com/99720107/196673468-079c83bf-48ab-4115-b2d2-af3fe4e85b1d.png)

Reply:
https://developer.android.com/training/location/geofencing#define-a-broadcast-receiver-for-geofence-transitions

Reply:
and does it work now? or what is the situation now?

Reply:
also you changed way more than i was asking for and the pendingintent already existed here https://github.com/cleverpush/cleverpush-android-sdk/pull/85/commits/d53e6baa6e9f3fa8da9bba5af0928f9d15cb9168#diff-dadb76999f5ed7d5f6c363d011ef4b354183a8b5cee4ecd118f2152c882891f1R1068
please check with manav if you didnt yet